gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat Nov 02 21:55:43 CET 2013
+#Mon Mar 03 09:05:14 JST 2025
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-bin.zip
```

Comment:
I upgraded Gradle to build with the new OpenJDK.
The old OpenJDK does not support Apple Silicon, making development difficult.